.github/renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended"
+    ],
+    "configMigration": true,
+    "lockFileMaintenance": {
+        "enabled": true,
+        "automerge": true,
+        "automergeType": "branch",
+        "ignoreTests": true
```

Comment:
Just to make sure: Are these fields of the `lockFileMaintenance` object, or are they fields of the root object?

Reply:
These are config options which are only set for keeping lockfiles up to date. `lockFileMaintenance` allows you to override global config keys for the lockfiles part.